UI/DamageIndicator.cs

```diff
@@ -0,0 +1,39 @@
+using Godot;
+
+
+public class DamageIndicator : TextureRect
+{
+	Vector2 ShotFirePosition;
+	float OriginalRotation;
```

Comment:
Can we get a more descriptive/accurate variable name? (If you think that this is already a good name feel free to explain why)

Two blank lines between members and the first function (unless is a constructor)